private/bufpkg/bufplugin/bufpluginconfig/testdata/success/go/buf.plugin.yaml

```diff
@@ -1,6 +1,8 @@
 version: v1
-name: buf.build/grpc/go
-plugin_version: v1.5.0
+name: buf.build/library/go-grpc
+plugin_version: v1.2.0
+deps:
+  - buf.build/library/go:v1.28.0-0
```

Comment:
Changed this to match what we might end up using as the naming convention (`protoc-gen-go-grpc` -> `go-grpc` plugin name) and some version numbers that track upstream.